scenarioo-client/app/manage/generalSettings/generalSettings.html

```diff
@@ -65,7 +65,7 @@ <h2>Full Text Search</h2>
             <div class="col-lg-12">
                 Full text search is <span class="sc-service-disabled">disabled</span>. <br/>
```

Comment:
For some reason this text is not visible here: http://demo.scenarioo.org/scenarioo-feature-675-versioning-of-docu/#/manage?branch=wikipedia-docu-example&build=last%20successful&comparison=Disabled&tab=configuration

Reply:
Not understand - of course this is not visible there, because it is NOT disabled. Was always like that, I think.

Reply:
Ah I see, I thought we also show a message when the feature is "enabled". So it's all fine.